src/include/api/vamana_index.h

```diff
@@ -58,6 +58,11 @@
  *   - An add method
  *   - A query method
  *
+ * We support all combinations of the following types for feature, id, and px
+ * datatypes:
+ *   - feature_type: uint8 or float
+ *   - id_type: uint32 or uint64
+ *   - px_type: uint32 or uint64
```

Comment:
vamana doesn't use a "px" datatype.  I used the name "px" in the context of IVF to be distinct from "index" because "index" is quite overloaded -- "px" means "partitioning index' -- i.e., the array that demarcates the begin and end of each partition in IVF. There is an analogous thing in `vamana_index` -- the type used as an indexing array that is part of vamana's graph.  In vamana's metadata it's called `adjacency_row_index_type` / `adjacency_row_index_datatype` -- so that should be used instead of `px_type` / `px_datatype`.

Reply:
Great, thanks, will update to that instead.